lang/cs/Org.Apache.REEF.IMRU.Tests/TestEvaluatorManager.cs

```diff
@@ -129,7 +129,7 @@ public void TestResetFailedEvaluators()
         [Fact]
         public void TestReachedMaximumNumberOfEvaluatorFailures()
         {
-            var evalutorManager = CreateTestEvaluators(3, 2);
+            var evalutorManager = CreateTestEvaluators(3, 1);
```

Comment:
We've changed logic of `ReachedMaximumNumberOfEvaluatorFailures` method, so this test is updated accordingly.